core/src/main/scala/com/salesforce/op/stages/impl/feature/SmartTextVectorizer.scala

```diff
@@ -189,8 +201,8 @@ private[op] case class TextStats
   val lengthMean: Double = lengthCounts.foldLeft(0.0)((acc, el) => acc + el._1 * el._2) / lengthSize
   val lengthVariance: Double = lengthCounts.foldLeft(0.0)(
     (acc, el) => acc + el._2 * (el._1 - lengthMean) * (el._1 - lengthMean)
-  )
-  val lengthStdDev: Double = math.sqrt(lengthVariance / lengthSize)
+  ) / lengthSize
```

Comment:
can `lengthSize` be `0`?

Reply:
Yep, and I think this is the behavior we want (these derived quantities being Double.NaN in this case).

The aggregations are already on the maps themselves, so these results being NaN will not propagate through - they are just descriptives. In the unlikely case where an entire column consists of stop words or something that doesn't tokenize (so lengthSize = 0), then we won't trigger removal either since Double.NaN < minLengthStdDev will always be false.

I can add a test where we check the values when lengthSize is 0 to make it more explicit.

Reply:
Ah, good point on NaN.